inventory/host_vars/ocp2.yaml

```diff
@@ -6,17 +6,25 @@ cluster_type: classic
 cluster_api_vip: 10.32.105.42
 cluster_ingress_vip: 10.32.105.49
 
+cluster_color: #0FF
+
 # Default is in roles/cluster/defaults/main.yaml - Override if needed:
 #openshift_version: 4.17.12
-stormshift_cluster_features: []
+stormshift_cluster_features:
+  - redhat-internal-certificate
+  - coe-sso
+  - look-and-feel
+  - lvms
```

Comment:
Just for information, that only applies to local execution. I assume the stormshift_cluster_features will be overwritten from the job template. 

We could set the default of the template to the value of stormshift_cluster_features

https://github.com/stormshift/automation/blob/69863ff538d7a7bcd8c9013a1aee2bdf77732039/configure-job-templates.yaml#L111